src/snowflake/snowpark/dataframe.py

```diff
@@ -4732,6 +4732,7 @@ def row_to_string(row: List[str]) -> str:
             + line
         )
 
+    @publicapi
```

Comment:
this is not a public api, the leading `_` indicates it is a private API - could you remove the decorator?